### ISSUE TITLE
Plots not shown in .Rmd repository

### ISSUE THREAD

**Issue by GenevieveDavis** [jdunic/Intro-To-Practical-Computing-R#10]
Are our plots supposed to show up on our document page when we push our .Rmd documents to github?  Mine aren't showing, only the text and code.  I tried pushing the knitted html document as well, which definitely doesn't work as it just shows garbled html code.
Thanks!

**Comment by bseitz234** [comment]
I had the same issue with my graphs not showing up as well. I guess changing the output to pdf would probably allow it to show up on github, but I think anyone who wants to see the graphs can download the .Rmd and knit it themselves?

**Comment by jdunic** [comment]
Yes, don't worry about the plots not showing up because we are mostly interested in the code and I will render them as necessary, or use the tool I will link to below. 

If you would like to view the rendered html files, you can view them using this website. I have tested this though and it doesn't seem to work for private repositories. 
[http://rawgit.com/](http://rawgit.com/)

I think you are right @bseitz234 that you can publish the files as pdfs and then the rendered files can be viewed on github. If you'd prefer to render pdfs that's great too!
